cns/service/main.go

```diff
@@ -880,6 +884,43 @@ func main() {
 		}
 	}
 
+	if cnsconfig.EnableSwiftV2 && cnsconfig.EnableK8sDevicePlugin {
+		initialVnetNICCount := 0
+		initialIBNICCount := 0
```

Comment:
If we're not writing to these, and assigning them off to appease the magic number linter, they should probably be consts.

Reply:
Updated

Reply:
done